### ISSUE TITLE
You plagiarised my ReadMe from my NSFW project, so what else have you stolen?

### ISSUE THREAD

**Issue by TechnikEmpire** [zeusees/HyperNSFW#2]

**Comment by lsy17096535** [comment]
You plagiarised the name from yahoo open_nsfw project, [so](https://github.com/yahoo/open_nsfw) what else have you stolen?

**Comment by TechnikEmpire** [comment]
Lol commented and then deleted it. NSFW is not Yahoo's copyright. Copying my unique code, models and text that are my copyright is called theft.

You're a thief and I'll be getting your repo DMCA'd here shortly.

**Comment by mattzhang1984** [comment]
Please show your fucking unique code which can prove that we plagiarised from your project. 
The two are completely different.
Our model is the MobileNet backbone and trained with the data we collected .
It is totally different from your code.
This task just a classifier. No need to copy your stupid code.

**Comment by TechnikEmpire** [comment]
So you didn't need to steal code but you needed to steal my readme? That sure makes a lot of sense. I'm supposed to believe you can write your own code when you can't even write your own plain readme lmfao.

**Comment by TechnikEmpire** [comment]
Thieves

**Comment by mattzhang1984** [comment]
I can prove where we steal from your fucking stupid readme?

**Comment by TechnikEmpire** [comment]
No thanks I don't feel like pointing you guys to exactly where you directly copied and pasted paragraphs from, you already know. You even borrowed my solver parameters and swapped weight decay and lr around cause you're a bunch of thieving clowns that don't actually know what you're doing. You're just stealing from people in the world and claiming work that isn't your own.

Your model is shit by the way. You should be hitting ~95% but instead you're hitting ~65% cause that's what happens when you mimic and steal instead of innovate and learn.

**Comment by lsy17096535** [comment]
So you're really good at training a squeezenet classifier  🌶

**Comment by TechnikEmpire** [comment]
> So you're really good at training a squeezenet classifier

I'm really good at not being a 2-bit thief stealing other peoples copyright and pretending its mine. That's something none of you can say. Enjoy life being bottom feeding imitators.

**Comment by mattzhang1984** [comment]
Yes, you good at not being a 2-bit thief.  but you really especially enjoy the joy of framing others.

**Comment by TechnikEmpire** [comment]
> Yes, you good at not being a 2-bit thief. but you really especially enjoy the joy of framing others.

Framing? This must be a language barrier. You did what you did. Suck it up and admit it, stop acting like babies. You both immediately knew me and my project because you spent time trawling my work and stealing from it. 

I'm flattered really, but you chose the wrong person to rip off. My squeezenet model is trash, was the first model I ever made and my results were polluted by bias. This is why your model turned out to be complete trash too. So while you're thieves, its to your own detriment. lmao
